src/components/ProfileInfos/index.js

```diff
@@ -0,0 +1,179 @@
+import Grid from '@material-ui/core/Grid';
+import { Patten } from "../../shared/styles/LoginStyles";
+import { Hidden } from "@material-ui/core";
+import { Button, makeStyles, Checkbox } from '@material-ui/core';
+import profileImage from "../../assets/images/profile_photo.svg";
+import Typography from '@material-ui/core/Typography';
+import { InputWrapper, LabelWrapper } from '../../shared/styles/ProfileInfosStyles';
+
+const useStyles = makeStyles(theme => ({
+    paper: {
+        width: '90  %',
+        position: 'relative',
+        height: 'auto',
+        padding: theme.spacing(1.4),
+        marginTop: 160,
+        marginBottom: 20,
+        '@media (max-width:684px)': {
+            width: '100%'
+        },
+        '@media only screen and (min-device-width: 768px) and (max-device-width: 1024px) and (orientation: portrait) and (-webkit-min-device-pixel-ratio: 2)': {
+            height: '80vh',
+            paddingTop: 150
+        }
+    },
+    title: {
+        color: '#726C6C',
+        fontWeight: 800,
+        fontFamily: `'Poppins', sans-serif;`,
+        padding: theme.spacing(1.4)
+    },
+    paragraph: {
+        paddingTop: theme.spacing(4),
+        paddingRight: 100,
+        paddingLeft: theme.spacing(4),
+        '@media (max-width:500px)': {
+            paddingRight:0,
+        },
+    },
+    img: {
+        width: 150,
+        height: 150,
+        marginTop: theme.spacing(4),
+        marginBottom: 20
+    },
+    profile: {
+        paddingLeft: theme.spacing(8),
+        '@media (max-width:850px)': {
+            paddingLeft: 'unset',
+            margin: 'auto'
+        },
+        '@media (max-width:600px)': {
+            alignItems: 'center'
+        }
+    },
+    button: {
+        height: 25,
+        width: 110,
+        textTransform: 'capitalize',
+        background: '#2196F3',
+        marginLeft: theme.spacing(1.4)
+    },
+    form: {
+        display: 'flex',
+        flexDirection: 'column',
+        marginLeft: 120,
+        '@media(max-width: 1111px)': {
+            marginLeft: 50
+        },
+        '@media(max-width: 320px)': {
+            marginLeft: 0
+        },
+    },
+    consent: {
+        paddingLeft: theme.spacing(8),
+        paddingRight: theme.spacing(8),
+        '@media(max-width: 600px)': {
+            paddingLeft: theme.spacing(0)
+        },
+    
+    },
+    confirm: {
+        height: 25,
+        width: 150,
+        textTransform: 'capitalize',
+        background: '#2196F3',
+        alignSelf: 'center'
+    },
+    span: {
+        fontFamily: `'Poppins', sans-serif;`,
+        fontWeight: 'bold'
+    }
+}))
+
+const ProfileInfos = () => {
+    const classes = useStyles();
+    
+    return (
+        <div style={{marginLeft: '5%', marginRight: '2%', marginBottom: 150 }}>
+            <Hidden only={['sm', 'xl', 'xs']}>
+                <Patten style={{position: 'absolute', right: 50, marginTop:'-55px'}}/>
+            </Hidden>
```

Comment:
Could you reduce the right margin so that the pattern and profile Paper appear as two separate elements? Because since it is beneath the paper it doesn't look nice. Consider the design that there is a certain part which is not beneath the paper element.

Reply:
Yeah, let me do that.

Reply:
Well done!